--- origsrc/gok-1.4.0/configure.in	2008-07-14 13:22:25.000000000 -0500
+++ src/gok-1.4.0/configure.in	2008-07-16 22:16:36.359375000 -0500
@@ -15,18 +15,6 @@
 AC_SUBST(GOK_INTERFACE_AGE)
 AC_SUBST(GOK_BINARY_AGE)
 
-# libtool versioning
-LT_RELEASE=$GOK_MAJOR_VERSION.$GOK_MINOR_VERSION
-LT_CURRENT=`expr $GOK_MICRO_VERSION - $GOK_INTERFACE_AGE`
-LT_REVISION=27
-LT_AGE=`expr $GOK_BINARY_AGE - $GOK_INTERFACE_AGE`
-LT_VERSION_INFO='-version-info ${LT_CURRENT}:${LT_REVISION}:${LT_AGE}'
-AC_SUBST(LT_VERSION_INFO)
-AC_SUBST(LT_RELEASE)
-AC_SUBST(LT_CURRENT)
-AC_SUBST(LT_REVISION)
-AC_SUBST(LT_AGE)
-
 dnl Specify a header configuration file
 AM_CONFIG_HEADER(config.h)
 
@@ -57,18 +45,18 @@
 AC_PATH_XTRA
 
 have_xinput=
-AC_CHECK_LIB(Xi, XOpenDevice, XINPUT_LIBS=-lXi)
+AC_CHECK_LIB(Xi, XOpenDevice, XINPUT_LIBS="-lXi -lXext -lX11")
 if test "x$XINPUT_LIBS" = x; then
   save_LIBS="$LIBS"
   for xinputpath in $x_libraries /usr/X11R6/lib /usr/openwin/lib; do
-    LIBS="-L$xinputpath -lXi"
+    LIBS="-L$xinputpath -lXi -lXext -lX11"
     case "$host" in
     *solaris*) XINPUT_RPATH_FLAGS="-R$xinputpath" ;;
     esac
     AC_MSG_CHECKING(for -lXi in $xinputpath)
-    AC_TRY_LINK([], [XOpenDisplay()], [
+    AC_TRY_LINK([], [XOpenDevice()], [
 	AC_MSG_RESULT(yes)
-	XINPUT_LIBS="$XINPUT_RPATH_FLAGS -L$xinputpath -lXi"
+	XINPUT_LIBS="$XINPUT_RPATH_FLAGS -L$xinputpath -lXi -lXext -lX11"
         LIBS="$save_LIBS"
 	break],[AC_MSG_RESULT(no)])
   done
@@ -76,7 +64,10 @@
     AC_MSG_ERROR(Couldn't find the XInput library. Check config.log for details)
   fi			       
 fi
+save_CFLAGS="$CFLAGS"
+CFLAGS="$X_CFLAGS $CFLAGS"
 AC_CHECK_HEADER(X11/extensions/XInput.h, have_xinput=yes)
+CFLAGS="$save_CFLAGS"
 if test "x$have_xinput" = xyes; then
    AC_DEFINE(HAVE_XINPUT)
 fi
--- origsrc/gok-1.4.0/gok/Makefile.am	2008-07-14 10:53:31.000000000 -0500
+++ src/gok-1.4.0/gok/Makefile.am	2008-07-16 22:16:36.375000000 -0500
@@ -26,8 +26,7 @@
 	$(GOK_CFLAGS)
 
 LDADD = $(GOK_LIBS)\
-	$(X_LIBS) \
-	@LT_VERSION_INFO@
+	$(X_LIBS)
 
 gok_LDFLAGS = -export-dynamic
 
